client/my-sites/email/email-providers-comparison/email-provider-details/email-provider-feature.jsx

```diff
@@ -2,13 +2,15 @@
  * External dependencies
  */
 import React from 'react';
-import Gridicon from 'components/gridicon';
+import Gridicon from 'calypso/components/gridicon';
 
 export default ( { title } ) => {
+	const deorphanedTitle =
+		'string' === typeof title ? title.replace( /([^ ]) ([^ ]+)$/, '$1\u00a0$2' ) : title;
```

Comment:
@akirk: not urgent but I was wondering what you think of this approach, and if there was already a function somewhere to deal with orphans that we weren't aware of.

Reply:
I am going to ping @dlind1 for the question above -- Alex is out on sabbatical.

Reply:
@stephanethomas @daledupreez I was looking for the function myself and stumbled on this comment from last year. :-D

The helper function is here: https://github.com/Automattic/wp-calypso/blob/9a3fc0feba3b8ddef3c40a04085883aa32dfc2bd/client/lib/formatting/index.js#L5

Reply:
Thanks @simison! I will address this in https://github.com/Automattic/wp-calypso/pull/52517.